File: WebApplication/src/modules/application/application-four/models/application-four-attached-file.dto.ts
import { AttachedFile } from "src/infrastructure/models/attached-file.model";
import { ApplicationFourAttachedFileType } from "../enums/application-four-attached-file-type";

export class ApplicationFourAttachedFileDto {
  type: ApplicationFourAttachedFileType;
  file: AttachedFile;

  constructor(type: ApplicationFourAttachedFileType) {
    this.type = type;
  }
}

File: StudentCredit.Application/Applications/ApplicationsFour/Dtos/ApplicationFourAttachedFileDto.cs
using FileStorageNetCore.Models;
using StudentCredit.Data.Applications.ApplicationFour.Enums;

namespace StudentCredit.Application.Applications.ApplicationsFour.Dtos
{
    public class ApplicationFourAttachedFileDto
    {
        public ApplicationFourAttachedFileType Type { get; set; }

        public AttachedFile File { get; set; }
    }
}
